src/conditions/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component, Fragment } from 'react';
+import ReactMarkdown from 'react-markdown';
+import { Form } from '../';
+import { Warning } from '@ukhomeoffice/react-components';
+
+class Conditions extends Component {
+  constructor(options) {
+    super(options);
+    this.toggleEdit = this.toggleEdit.bind(this);
```

Comment:
Is there a reason to do it this way and not the arrow function way we achieve this in other places?

Reply:
yes, we don't have that babel plugin enabled in non-projects land :(

Reply:
Carry on then.

Reply:
I don't think we do anyway, we didn't before.. perhaps we do? Are these my hands?